packages/components/src/Link/Link.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode, forwardRef } from "react";
+import Clickable, { ClickableProps } from "../Clickable";
+
+type LinkHTMLElementProps = Omit<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  "disabled"
+>;
+
+export type LinkProps = LinkHTMLElementProps & {
+  as?: "a" | React.ComponentType<any>;
```

Comment:
Since Link is for `<a>` and Button is for `<button>`, can we get rid of the `as` prop?

I can't think of too many scenarios where we'd want to allow arbitrary components. Maybe a specific Bootstrap or React-router link?

Reply:
The react-router link is definitely the main usage I'm expecting here. I think we're going to need some way to apply the button/link designs to other components and elements.

@anniehu4 and I have talked a bit about whether we should drop the `as` from the `Button` and `Link` components and have people use `Clickable` instead (but maybe with a different name) so the `Link` and `Button` components' purpose is clearer since they both just apply the same styles anyway. Thoughts?

Reply:
Seems reasonable. Expose Clickable for arbitrary components (which if I understand correctly is exactly why it exists). And ensure Link and Button get you exactly those things.